packages/liveblocks-react/src/umbrella-store.ts

```diff
@@ -617,7 +618,7 @@ function createStore_forNotifications() {
   }
 
   function markAllRead(readAt: Date) {
-    return signal.mutate((lut) => {
```

Comment:
Was this `return` unnecessary?

Reply:
None of the other similar methods return it. It's a `void` anyway, so it doesn't matter.